### ISSUE TITLE
cfinterface and messaging

### ISSUE THREAD

**Issue by bdw429s** [foundeo/toscript#30]
Testing the CLI command on the ColdBox source code (which still has some tags floating around) I noticed that interfaces don't work. 
```
! Error: system\ioc\scopes\IScope.cfc
  Unable to convert tag: cfinterface to CFML Script
+  (Converted): system\ioc\scopes\IScope.cfc
```
Also, the messaging is confusing since it says it was unable to convert, but then says the file was converted (it wasn't).

**Comment by pfreitag** [comment]
Thanks -- yeah that is confusing I guess, it was unable to convert the `<cfinterface>` tag but it was probably able to convert any other CFML tags in there (like cffunction, etc) so it does not fail fully, it just wraps the tags it cannot convert in a comment.

**Comment by bdw429s** [comment]
Yeah, in that case it was the entire CFC :)  I'd say just swap the green "success" message to something like a yellow "partially converted, please review" message.  My eyes were scanning for green lines at first thinking they were the ones that were 100% good to go.
